dashboards.tf

```diff
@@ -20,26 +20,10 @@ resource "dynatrace_dashboard_sharing" "signin-signup" {
   }
 }
 
-resource "dynatrace_json_dashboard" "slos" {
-  contents = file("${path.module}/dashboards/slos.json")
-}
-
-resource "dynatrace_dashboard_sharing" "slos" {
-  dashboard_id = dynatrace_json_dashboard.slos.id
-
-  enabled = true
-
-  permissions {
-    permission {
-      level = "VIEW"
-      type  = "ALL"
-    }
-    permission {
-      id    = data.dynatrace_iam_group.all.id
-      level = "VIEW"
-      type  = "GROUP"
-    }
-  }
+module "slo_dashboard" {
+  count  = local.is_production ? 1 : 0
+  source = "./modules/dashboard"
+  path   = "slos.json"
```

Comment:
Assuming this resolves to the correct file path

Reply:
https://github.com/govuk-one-login/observability-configuration/blob/096eb1dc68d35868bcd7c603f2efea75507bedf1/modules/dashboard/main.tf#L2 

checks out, as long as terraform is executed in the root of this project it should pickup the right file

Reply:
It's all automated via the GitHub actions, but yes you're right Sam, this particular dashboard is in the root of the dashboards folder and hence why the "path" is only the name of the dashboard json file.